backend/Squirrel.Core/Squirrel.Core.BLL/Interfaces/IAuthService.cs

```diff
@@ -1,9 +1,11 @@
 ﻿using Squirrel.Core.Common.DTO.Auth;
+using Squirrel.Core.Common.DTO.Users;
 
 namespace Squirrel.Core.BLL.Interfaces;
 
 public interface IAuthService
 {
     Task<RefreshedAccessTokenDto> LoginAsync(UserLoginDto userLoginDto);
     Task<RefreshedAccessTokenDto> RegisterAsync(UserRegisterDto userRegisterDto);
+    Task<AuthUserDTO> AuthorizeWithGoogle(GoogleToken googleToken);
```

Comment:
it is async method, so it's better to add 'Async' suffix

Reply:
added Async